x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshots.java

```diff
@@ -304,25 +280,17 @@ public void onIndexModule(IndexModule indexModule) {
         IndexNameExpressionResolver indexNameExpressionResolver,
         Supplier<DiscoveryNodes> nodesInCluster
     ) {
-        if (SEARCHABLE_SNAPSHOTS_FEATURE_ENABLED) {
-            return List.of(
-                new RestSearchableSnapshotsStatsAction(),
-                new RestClearSearchableSnapshotsCacheAction(),
-                new RestMountSearchableSnapshotAction(),
-                new RestRepositoryStatsAction()
-            );
-        } else {
-            return List.of();
-        }
+        return List.of(
+            new RestSearchableSnapshotsStatsAction(),
+            new RestClearSearchableSnapshotsCacheAction(),
+            new RestMountSearchableSnapshotAction(),
+            new RestRepositoryStatsAction()
+        );
     }
 
     @Override
     public Map<String, ExistingShardsAllocator> getExistingShardsAllocators() {
-        if (SEARCHABLE_SNAPSHOTS_FEATURE_ENABLED) {
-            return Collections.singletonMap(SearchableSnapshotAllocator.ALLOCATOR_NAME, new SearchableSnapshotAllocator());
-        } else {
-            return Collections.emptyMap();
-        }
+        return Collections.singletonMap(SearchableSnapshotAllocator.ALLOCATOR_NAME, new SearchableSnapshotAllocator());
```

Comment:
nit: use `Map.of()`

Reply:
Have changed it, but don't think it's a great use of my time using these features if I have to change everything back on  backport.

Reply:
That was more for the sake of coherency - either use one or the other but avoid mixing them